Makefile

```diff
@@ -55,6 +55,21 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: cma-check-all-csv
+cma-check-all-csv: build-testutil ## Verify that CMA CSV files look right
+	hack/cma-check-all-csv.sh
+
+.PHONY: build-testutil
+build-testutil: bin/yaml2json bin/json2yaml ## Build utilities needed by tests
+
+# utilities needed by tests
+bin/yaml2json: cmd/testutil/yaml2json/yaml2json.go
+	mkdir -p bin
+	go build $(GOGCFLAGS) -ldflags "$(LD_FLAGS)" -o bin/ "github.com/kedacore/keda-olm-operator/cmd/testutil/yaml2json"
+bin/json2yaml: cmd/testutil/json2yaml/json2yaml.go
+	mkdir -p bin
+	go build $(GOGCFLAGS) -ldflags "$(LD_FLAGS)" -o bin/ "github.com/kedacore/keda-olm-operator/cmd/testutil/json2yaml"
```

Comment:
Is this doable during the generation of the CI image used in the build process? The tool won't change very often; it may save time/resources to have the binaries present in the image.

Reply:
Good idea, I'll look into it.

Reply:
On second thought, I think that since only one CI test will be using these tools, it's fine to just build them immediately before the test. If we had several CI tests using them, then maybe we could have the CI tests create a `srcplus` image that is based on the `src` image, but build the common tools needed by various CI tests running on the `src` image, but it seems like it would increase CI testing latency and use more resources to build the intermediate image where only one test would use it. But I'll keep the idea in mind in case we add more tests that use these tools.

Reply:
Agreed.